### PR TITLE
removes specifically the heart attack effect from being fat

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# Document the changes in your pull request

technically this is funny but it actually kills people way more often than it should just because it's a MASSIVE pain to deal with obesity without the help of a chemist or bartender or actual doctor who knows what they are doing (rare), coupled with how easy it is to accidentally overeat (especially if you don't know it's lethal) and get stuck with several minutes of fatness a feature that randomly kills you for going overweight is kind of stupid

I'm keeping the heart damage though that's thematic and still a good downside since you lose a large amount of time post death where you can be defibbed without a bypass or replacement heart

# Changelog
:cl:  
rscdel: being fat no longer causes heart attacks on its own, it will still cause massive damage to the heart over time
/:cl:

### CODE REVIEW

**Comment by ChesterTheCheesy** [comment]
bro just zap yourself on an airlock if you're getting heart attacked

**Comment by maxion12345** [comment]
Wasn't it decided back in march when you tried this (https://github.com/yogstation13/Yogstation/pull/11057) that instead of removing the mechanic, it would be better to improve upon it by perhaps making the gym holodeck more functional? The gym machines already prevent heart attacks if you exercise. Making it so you can also lose fat by exercising instead of just preventing the effects of being fat would work. We keep the penalty for ignoring your potential heart failure, and the players have a way to remove being fat without asking for surgery, chemicals, or that one obscure fat removal machine nobody builds

tl;dr last time this was posted it was collectively agreed that outright removal was bad, and a better option might be increasing functionality of the weights in the holodeck

**Comment by JamieD1** [comment]
Would prefer to add more ways to reduce fat instead of removing the downsides from it.
